hoomd/simulation.py

```diff
@@ -236,12 +236,17 @@ def operations(self, operations):
                     "Cannot add `hoomd.Operations` object that belongs to "
                     "another `hoomd.Simulation` object.")
             # Switch out `hoomd.Operations` objects.
-            elif self._operations._scheduled:
+            reschedule = False
+            if self._operations._scheduled:
                 self._operations._unschedule()
-                self._operations._simulation = None
-                operations._simulation = self
-                operations._schedule()
-                self._operations = operations
+                reschedule = False
```

Comment:
```suggestion
                reschedule = True
```
I think this will fix some of those failing tests.